src/index.js

```diff
@@ -228,41 +228,58 @@ class App extends React.Component {
           onMouseDown={e => {
             const x = e.clientX - e.target.offsetLeft;
             const y = e.clientY - e.target.offsetTop;
-            const element = newElement(this.state.elementType, x, y);
 
-            if (this.state.elementType === "text") {
-              const text = prompt("What text do you want?");
-              if (text === null) {
-                return;
-              }
-              element.text = text;
-              element.font = "20px Virgil";
-              const font = context.font;
-              context.font = element.font;
-              element.measure = context.measureText(element.text);
-              context.font = font;
-              const height =
-                element.measure.actualBoundingBoxAscent +
-                element.measure.actualBoundingBoxDescent;
-              // Center the text
-              element.x -= element.measure.width / 2;
-              element.y -= element.measure.actualBoundingBoxAscent;
-              element.width = element.measure.width;
-              element.height = height;
-            }
+            if (this.state.elementType === "selection") {
+              const selectedElement = elements.find(el => {
```

Comment:
Same comment as the other pull request, we should be consistent in how we name variables if possible. I used “element” everywhere else so far. Happy to see it changed to something else

Reply:
I usually avoid shadowed variable declarations,`element` is already declared in the upper scope

Reply:
Oh I see. It’s probably a sign that it’s time to refactor some of this huge function in some way :)

Maybe having a helper for finding selected elements would be useful.

Anyway, this doesn’t need to be done in this PR.